examples/benchmark/nc/docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+version: '2.1'
+
+volumes:
+  files:
+    driver: local
+  mysql:
+    driver: local
+  backup:
+    driver: local
+  redis:
+    driver: local
+
+services:
+  nextcloud:
+    image: nextcloud:19.0.0
+    restart: always
+    ports:
+      - 8080:80
+    depends_on:
+      - db
+      - redis
+    environment:
+      - MYSQL_ROOT_PASSWORD=nextcloud
+      - MYSQL_HOST=db
+      - MYSQL_PASSWORD=nextcloud
+      - MYSQL_DATABASE=nextcloud
+      - MYSQL_USER=nextcloud
+      - NEXTCLOUD_ADMIN_USER=admin
+      - NEXTCLOUD_ADMIN_PASSWORD=admin
```

Comment:
also needs `NEXTCLOUD_TRUSTED_DOMAINS=myip`
should be dynamic like `OCIS_DOMAIN`